mne/preprocessing/tests/test_ica.py

```diff
@@ -214,6 +214,20 @@ def test_ica_noop(n_components, n_pca_components, tmpdir):
     _assert_ica_attributes(ica)
 
 
+@pytest.mark.parametrize("method, max_iter_default", [("fastica", 1000),
+                         ("infomax", 500), ("picard", 500)])
+def test_ica_max_iter_(method, max_iter_default):
+    """Test that ICA.max_iter is set to the right defaults."""
+    _skip_check_picard(method)
+
+    # check that default max_iter comes out for no input
+    ica = ICA(n_components=3, method=method)
+    assert ica.max_iter == max_iter_default
+    # check that user input comes out unchanged
+    ica = ICA(n_components=3, method=method, max_iter=2000)
```

Comment:
how fast is this test? any chance to make it faster using super high tolerance?

Reply:
No ICA is fitted in this little test, it just asserts that the value in the class parameter is set right. So timing is fast (0.3 s), and I wouldn't know how to speed it up, as the values in n_components or max_iter don't make a difference at this point...(?)

Reply:
ok